src/options/view.rs

```diff
@@ -350,12 +352,17 @@ impl TimeFormat {
                 //   - there is nothing after `+`
                 // line 1 will be empty when:
                 //   - `+` is followed immediately by `\n`
-                let empty_non_recent_format_msg = "Custom timestamp format is empty, \
-                    please supply a chrono format string after the plus sign.";
-                let non_recent = lines.next().expect(empty_non_recent_format_msg);
+                let invalid_non_recent_format_msg = "Custom timestamp format is invalid, \
```

Comment:
This would be more appropriate as a const, I know it wasn't introduced in this PR but it would make a lot of sense to fix it while here